docs/cortex/installation-and-configuration/run-cortex-with-docker.md

```diff
@@ -1,72 +1,112 @@
-| `--no-config`        | `no_config=1`        | Do not configure Cortex                           |
-| `--no-config-secret` | `no_config_secret=1` | Do not add the random secret to the configuration |
-| `--no-config-es` | `no_config_es=1` | do not add elasticsearch hosts to configuration
-| `--es-uri <uri>` | `es_uri=<uri>` | use this string to configure elasticsearch hosts (format: http(s)://host:port,host:port(/prefix)?querystring)
-| `--es-hostname <host>` | `es_hostname=host` | resolve this hostname to find elasticsearch instances
-| `--secret <secret>` | `secret=<secret>` | secret to secure sessions
-| `--show-secret` | `show_secret=1` | show the generated secret
-| `--job-directory <dir>` | `job_directory=<dir>` | use this directory to store job files
-| `--docker-job-directory <dir>` | `docker_job_directory=<dir>` | indicate the job directory in the host (not inside container)
-| `--analyzer-url <url>` | `analyzer_urls=<url>,<url>,...` | where analyzers are located (url or path)
-| `--responder-url <url>` | `responder_urls=<url>,<url>,...` | where responders are located (url or path)
-| `--start-docker` | `start_docker=1` | start an internal docker (inside container) to run analyzers/responders
-| `--daemon-user <user>` | `daemon_user=<user>` | run cortex using this user
-
-At the end of the generated configuration, the file `/etc/cortex/application.conf` is included. Thus you can override any setting by binding your own `application.conf` into this file:
+| `--no-config`        | `no_config=1`        | Don't configure Cortex                           |
+| `--no-config-secret` | `no_config_secret=1` | Don't add the random secret to the configuration |
+| `--no-config-es` | `no_config_es=1` | Don't add Elasticsearch hosts to the configuration
+| `--es-uri <uri>` | `es_uri=<uri>` | Configure Elasticsearch hosts with this string (format: http(s)://host:port,host:port(/prefix)?querystring)
+| `--es-hostname <host>` | `es_hostname=host` | Resolve this host name to locate Elasticsearch instances
+| `--secret <secret>` | `secret=<secret>` | Secret used to secure sessions
+| `--show-secret` | `show_secret=1` | Display the generated secret
+| `--job-directory <dir>` | `job_directory=<dir>` | Directory to store job files
+| `--docker-job-directory <dir>` | `docker_job_directory=<dir>` | Host directory corresponding to the job directory (outside the container)
+| `--analyzer-url <url>` | `analyzer_urls=<url>,<url>,...` | URLs or paths where analyzers are located
+| `--responder-url <url>` | `responder_urls=<url>,<url>,...` | URLs or paths where responders are located
+| `--start-docker` | `start_docker=1` | Start an internal Docker daemon inside the container to run analyzers/responders
+| `--daemon-user <user>` | `daemon_user=<user>` | Run Cortex using this user
+
+For detailed information on all Docker parameters and advanced configuration options, see [Parameters for Docker](parameters-docker.md).
```

Comment:
I'm afraid there's an ambiguity in this part, we're mixing up:
- Parameters (flags) used to customize the configuration auto-generated by Cortex at startup (all `--(...)` and/or env vars)
- Config available in the `application.conf` to tweak Cortex configuration

In the end, env vars and Cortex flags are translated in the Cortex `application.conf` file thanks to the entrypoint, but we may want to clarify a bit this sentence.